src/tests/ftest/control/daos_agent_support_collect_log.py

```diff
@@ -0,0 +1,53 @@
+"""
+  (C) Copyright 2023 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+from support_test_base import SupportTestBase
+
+
+class DaosAgentSupportCollectLogTest(SupportTestBase):
+    # pylint: disable=too-many-ancestors
+    """Test Class Description:
+            Verify the daos_server support collect-log command.
+
+    :avocado: recursive
+    """
+
+    def test_daos_agent_collect_log(self):
+        """JIRA ID: DAOS-10625
+
+        Test Description:
+            Test daos_agent support collect-log command completes successfully.
+
+        :avocado: tags=all,full_regression
+        :avocado: tags=hw,medium
+        :avocado: tags=control,basic,support,daos_server
+        :avocado: tags=test_daos_agent_support_collect_log
```

Comment:
Please add the class name tag as well
```suggestion
        :avocado: tags=DaosAgentSupportCollectLogTest,test_daos_agent_support_collect_log
```